google_maps/public/html/google_maps_edit.html

```diff
@@ -1,15 +1,67 @@
-<div class="wrapper-comp-settings is-active editor-with-buttons google-maps-edit-wrapper">
-  <p>Welcome to Google Maps Xblock!</p>
+{% load i18n %}
+
+<div class="wrapper-comp-settings is-active editor-with-buttons" id="google-maps-edit-wrapper">
+  <div class="xblock-inputs editor_content_wrapper">
+    <ul class="list-input settings-list">
+      <li class="field comp-setting-entry is-set edit-display-name-container">
+        <div class="wrapper-comp-setting">
+          <label class="label setting-label" for="edit_display_name">{{ fields.display_name.display_name }}</label>
+          <input class="input setting-input edit-display-name" id="edit_display_name" value="{{ self.display_name }}" type="text" 
+          data-default-value="{{ fields.display_name.default }}">
+          <button class="action setting-clear clear-display-name" type="button" name="setting-clear">
+            <i class="icon fa fa-undo"></i>
+          </button>
+        </div>
+        <span class="tip setting-help">{{ fields.display_name.help }}</span>
+      </li>
+
+      <div class="google-maps-inputs">
+        <li class="field comp-setting-entry is-set">
+          <div class="wrapper-comp-setting">
+            <label class="label setting-label" for="edit_place_name">{{ fields.place_name.display_name }}</label>
+            <input class="input setting-input edit-place-name" id="edit_place_name" value="{{ self.place_name }}" type="text" 
+            placeholder="{% trans 'Enter a location' %}" data-default-value="{{ fields.place_name.default }}">
+          </div>
+        </li>
+
+        <li class="field comp-setting-entry is-set">
+          <div class="wrapper-comp-setting">
+            <label class="label setting-label" for="edit_iframe_width">{{ fields.iframe_width.display_name }}</label>
+            <input class="input setting-input edit-iframe-width" id="edit_iframe_width" value="{{ self.iframe_width }}" type="number" 
+            data-default-value="{{ fields.iframe_width.default }}">
+            <button class="action setting-clear clear-iframe-width" type="button" name="setting-clear">
+              <i class="icon fa fa-undo"></i>
+            </button>
```

Comment:
Same thing as for the button above.
`<span class="sr">{% trans "Reset iframe width to its default value" %}</span>`
and aria-hidden on the `i` element.